detectorSegmentations/include/detectorSegmentations/FCCSWHCalPhiRow_k4geo.h

```diff
@@ -74,6 +74,11 @@ namespace DDSegmentation {
      */
     void defineCellIndexes(const unsigned int layer) const;
 
+    /**  Find neighbours of the cell.
+     *   Implement the signature from the Segmentation base class.
+     */
+    virtual void neighbours(const CellID& cellID, std::set<CellID>& neighbours) const override;
+
```

Comment:
Super nitpicky here, but could we move this up to the other definition of `neighbours`, so that they appear next to each other? I was just wondering what the other signature was to check if it's only marginally of, or whether it's completely different (which it seems to be).